addons/mass_mailing/views/snippets_themes.xml

```diff
@@ -191,21 +191,21 @@
 </template>
 
 <template id="s_mail_block_footer_social" name="Footer Center">
-    <div class="s_footer_social o_mail_block_footer_social o_mail_footer_social_center o_mail_snippet_general bg-200">
+    <div class="s_footer_social o_mail_block_footer_social o_mail_footer_social_center o_mail_snippet_general bg-200 pt16">
         <div class="container">
             <div class="row">
-                <div class="col-lg o_mail_footer_social">
+                <div class="col-lg o_mail_footer_social" style="text-align: center;">
                     <t t-call="mass_mailing.social_links"/>
                 </div>
             </div>
             <div class="row">
-                <div class="col-lg o_mail_footer_links">
+                <div class="col-lg o_mail_footer_links" style="text-align: center;">
                     <a role="button" href="/unsubscribe_from_list" class="btn btn-link">Unsubscribe</a>
```

Comment:
@mart-e I felt like "it's not a big deal if people don't have this fix if they didn't update their views because it's only a styling change", then I figured: this is going to break the translations, isn't it ?

Reply:
@dmo-odoo if it was a mail template yes but here it's a view, so the translations are split and only "Unsubscribe" and "All Rights Reserved" are translated, so all good :+1: 
(and in case the PR did change the translations, don't forget to reexport the .pot file)